zip-0000.rst

```diff
@@ -4,7 +4,7 @@
   Title: ZIP Process
   Owners: Daira Hopwood
           Josh Cincinnati
-  Status: Draft
+  Status: Active
   Type: Process
```

Comment:
```suggestion
  Category: Process
```

Category is listed in the ZIP preamble so presumably this should be updated?

Reply:
Yup just updated in latest commit, good catch @garethtdavies!